fdb-relational-server/src/main/java/com/apple/foundationdb/relational/server/FRL.java

```diff
@@ -230,11 +232,14 @@ private static void addPreparedStatementParameter(RelationalPreparedStatement re
                 relationalPreparedStatement.setString(index, parameter.getParameter().getString());
                 break;
             case Types.BIGINT:
-                relationalPreparedStatement.setInt(index, parameter.getParameter().getInteger());
```

Comment:
You can't change `prepared.yamsql` without the client-side changes, right? So the test has to stay disabled. Have you run the latest client against this build?

Reply:
Yes. Enabling the test on the patch release would require the client side changes that would require changes to the `fob-relational-api` module. As discussed, it is not required at this point, so was left out of the back port.
I tested the changes locally with a manually built revision of this branch's server, where the `supported_version` was manually set (see below). Note that newer versions of the `prepared` test will not pass since we haven't ported the change to all of them.
<img width="590" alt="image" src="https://github.com/user-attachments/assets/bb0c7c2e-9538-418a-8987-c7ec8b1b86ef" />
<img width="590" alt="image" src="https://github.com/user-attachments/assets/1eabbba6-0c5c-488d-bbe5-859f60ed3d04" />